lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+const _ = require('lodash');
+const { addCustomResourceToService } = require('../../../../customResources');
+
+class AwsCompileEventBridgeEvents {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+    this.provider = this.serverless.getProvider('aws');
+
+    this.hooks = {
+      'package:compileEvents': this.compileEventBridgeEvents.bind(this),
+    };
+  }
+
+  compileEventBridgeEvents() {
+    const { service } = this.serverless;
+    const { provider } = service;
+    const { compiledCloudFormationTemplate } = provider;
+    const iamRoleStatements = [];
+
+    service.getAllFunctions().forEach(functionName => {
+      const functionObj = service.getFunction(functionName);
+      const FunctionName = functionObj.name;
+
+      if (functionObj.events) {
+        functionObj.events.forEach((event, idx) => {
+          if (typeof event.eventBridge === 'object') {
+            idx++;
```

Comment:
Is this just because we don't want `FuncName-rule-0`?

Reply:
Yes, exactly.